packages/website/src/content/docs/main/reference/infrastructure-modules/direct/authentik/authentik_github_sso/index.mdx

```diff
@@ -0,0 +1,160 @@
+3. Set the `acs_url` input using the `assertion consumer service URL` from above.
+4. Run `pf-tf-init`.
+5. Run `terragrunt apply`.
+6. Note the output as you'll use it in the following steps.
+
+### Complete GitHub SAML single sign-on
+
+1. Resume the Authentication security page
+2. Go to Security -> Authentication security.
+3. Set `Sign on URL` with the `sso_post_url` output value from above.
+4. Set `Issuer` with the `issuer_url` output value from above.
+5. Set `Public certificate` from the `certificate` output value from above.
+   ![SAML Form](doc\_images/github-saml-form.png)
+6. Click on `Test SAML configuration`.
+7. Save the `recovery codes` that you are prompted with.
+8. Click on `Save`.
+
+### Test and Turn on Require SAML SSO
+
+1. Go to your Authentik instance.
+2. Find the GitHub application. Ensure you are in the user dashboard, not the admin dashboard.
+   ![Github Application](doc\_images/github-application.png)
+3. Click and confirm that you are able to login.
+
+## Providers
+
+The following providers are needed by this module:
+
+- [authentik](https://registry.terraform.io/providers/goauthentik/authentik/2024.8.4/docs) (2024.8.4)
+
+- [kubectl](https://registry.terraform.io/providers/alekc/kubectl/2.1.3/docs) (2.1.3)
+
+- [kubernetes](https://registry.terraform.io/providers/hashicorp/kubernetes/2.34.0/docs) (2.34.0)
+
+- [random](https://registry.terraform.io/providers/hashicorp/random/3.6.3/docs) (3.6.3)
+
+- [tls](https://registry.terraform.io/providers/hashicorp/tls/4.0.6/docs) (4.0.6)
+
+## Required Inputs
+
+The following input variables are required:
+
+### acs\_url
+
+Description: The ACS url provided by GitHub when configuring an external identity provider
+
+Type: `string`
+
+### authentik\_domain
+
+Description: The domain name of the Authentik instance
+
+Type: `string`
+
+### organization\_name
+
+Description: The name of your organization
+
+Type: `string`
+
+## Optional Inputs
+
+The following input variables are optional (have default values):
+
+### authentik\_namespace
+
+Description: The kubernetes namespace where Authentik is deployed
+
+Type: `string`
+
+Default: `"authentik"`
+
+### extra\_allowed\_groups
+
+Description: Additional groups that can access GitHub
+
+Type: `set(string)`
+
+Default: `[]`
+
+### media\_configmap
+
+Description: The ConfigMap holding the static media that Authentik will use
+
+Type: `string`
+
+Default: `"media"`
+
+### ui\_description
+
+Description: The description to display in the Authentik web dashboard
+
+Type: `string`
+
+Default: `"GitHub"`
+
+## Outputs
+
+The following outputs are exported:
+
+### certificate
+
+Description: The certificate used to sign SAML responses
+
+### issuer\_url
+
+Description: The Authentik issuer URL for the GitHub provider
+
+### saml\_metadata
+
+Description: The SAML metadata for the GitHub provider. https://en.wikipedia.org/wiki/SAML_metadata
```

Comment:
Not a blocker, but for links you can use markdown format in the variable description so it isn't ugly like this.